install_plugin_linux.sh

```diff
@@ -17,3 +22,16 @@ else
         echo "Added user_allow_other added to $FUSE_CONF"
     fi
 fi
+
+### Install Cloudfuse
+echo "Installing Cloudfuse"
+apt-get install libssl-dev
+apt-get install ./cloudfuse*.deb
+
+### Install plugin
+echo "Installing Cloudfuse plugin"
+systemctl restart networkoptix-mediaserver.service
```

Comment:
Do you need to restart the server twice?

Reply:
Good catch. Actually, the first should stop, and then the second should start it. This should allow us to run the script if the plugin is already installed.